.github/workflows/ci-knip.yml

```
@@ -0,0 +1,37 @@
+concurrency: ${{ github.workflow }}-${{ github.ref }}
+
+permissions:
+  issues: write
+  repository-projects: read
+  contents: write
+  pull-requests: write
+  pages: write
+  id-token: write
+
+jobs:
+    lint:
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 2 but found 4

(indentation)